### PR TITLE
Add smoke tests for the service

### PR DESCRIPTION
**Pull request by lgonczar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-341

### Change description ###

Add smoke tests for the service

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/smokeTest/java/uk/gov/hmcts/reform/draftstore/client/DraftStoreClient.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        String idamToken,
+        String encryptionSecret
+    ) {
+        this.draftStoreUrl = draftStoreUrl;
+        this.s2sToken = s2sToken;
+        this.idamToken = idamToken;
+        this.encryptionSecret = encryptionSecret;
+    }
+
+    public String createDraft(String documentJson) throws JSONException, IOException {
```
Comment (rkondratowicz) [inline src/smokeTest/java/uk/gov/hmcts/reform/draftstore/client/DraftStoreClient.java]: add javadoc for return value

Comment (lgonczar) [reply src/smokeTest/java/uk/gov/hmcts/reform/draftstore/client/DraftStoreClient.java]: done

File: src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java [hunk 2]
```diff
@@ -1,25 +1,72 @@
...
+import java.util.Optional;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
 public class ApplicationSmokeTest extends SmokeTestSuite {
 
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private static final String SAMPLE_DRAFT_JSON = "{\"some\": \"draft\"}";
```
Comment (rkondratowicz) [inline src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java]: may be worth renaming to 'sample document' as this is not the actual draft json that's sent to the API.
or just create an instance of `CreateDraft` model here.

Comment (lgonczar) [reply src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java]: Fair point. Done.

**Pull request merged**